scrabble.js

```diff
@@ -1,16 +1,253 @@
-
 const Scrabble = {
-  score(word) {
-
+  score_per_char(){
```

Comment:
It makes more sense to have this as an object rather than a function which returns an object.